diff --git a/dist/runners/bun-process/runner.mjs b/dist/runners/bun-process/runner.mjs
index 69d0e35bae438e6a1ec77ed551062df77ad0529c..346d8085e6287c163e9448ced9d332f013fedc93 100644
--- a/dist/runners/bun-process/runner.mjs
+++ b/dist/runners/bun-process/runner.mjs
@@ -122,6 +122,8 @@ var BunProcessEnvRunner = class extends BaseEnvRunner {
 		child.on("message", (message) => {
 			this._handleMessage(message);
 		});
+		child.stdout?.pipe(process.stdout);
+		child.stderr?.pipe(process.stderr);
 		this.#process = handle;
 	}
 };
