diff --git a/dist/gridstack.js b/dist/gridstack.js
index da954f80e82a43096c91e42e40c764b2ebeaa17d..021383d53343934e6f17a6ae6b03a5be1a98c365 100644
--- a/dist/gridstack.js
+++ b/dist/gridstack.js
@@ -1742,7 +1742,13 @@ class GridStack {
             // order doesn't match, re-order DOM
             nodes.forEach(n => {
                 if (n.el && n.el.parentElement === this.el) {
-                    this.el.appendChild(n.el);
+                    // Preserve iframe documents and focus while sorting existing widgets.
+                    if (this.el.moveBefore) {
+                        this.el.moveBefore(n.el, null);
+                    }
+                    else {
+                        this.el.appendChild(n.el);
+                    }
                 }
             });
         }
